src/api/UsersAPI.js

```diff
@@ -1,5 +1,6 @@
 import APIClient from "./APIClient";
 import Cookies from "universal-cookie";
+import { tsThisType } from "@babel/types";
```

Comment:
What is this for?

Reply:
I have no idea. Probably some vscode autocomplete autoimport BS. I'll remove it real quick